kubernetes/loculus/values.yaml

```diff
@@ -1763,6 +1763,6 @@ defaultResources:
     cpu: "20m"
   limits:
     memory: "1Gi"
-    cpu: "20m"
+    cpu: "1000m"
```

Comment:
huh yeah this seems an odd decision! I guess we accidentally added this! 🤔  we could just remove the limit entirely?

Reply:
```suggestion
```